tests/test_freshdesk_bookmarks_stream_with_fillter_param.py

```diff
@@ -0,0 +1,153 @@
+import tap_tester.connections as connections
+import tap_tester.runner as runner
+import tap_tester.menagerie as menagerie
+import dateutil.parser
+from datetime import timedelta
+from datetime import datetime as dt
+
+from base import FreshdeskBaseTest
+
+class BookmarkTest(FreshdeskBaseTest):
+    """Test tap sets a separate bookmark for PUBLISHED and SCHEDULED tweets and 
+    respects it for the next sync of a tweets stream"""
```

Comment:
Please update the above comment, It looks incorrect for this test.

Reply:
Updated